components/odh-notebook-controller/controllers/notebook_controller.go

```diff
@@ -18,11 +18,12 @@ package controllers
 import (
```

Comment:
There aren't any regression tests added as part of this change. Do you intend to create some later?

Reply:
There are, I am still checking how to adapt these changes on the e2e test

Reply:
In the second scenario, the certificates will become outdated because the notebook is being treated as already existing.

Reply:
Is there a way for notebook user to get the new certificates in that case?

Reply:
@harshad16 did you discuss this yesterday?

Reply:
I couldn't , i will check via slack.